README.md

```diff
@@ -27,7 +27,6 @@ Use the binaries from [the release page](https://github.com/snyk-tech-services/j
     -jiraProjectID=<12345>                                              // Jira project ID the tickets will be opened against
     -jiraTicketType=<Task|Bug|....>                                     // Optional. Type of ticket to open. Defaults to Bug
     -severity=<critical|high|medium|low>                                // Optional. Severity threshold to open tickets for. Defaults to low.
-    -priorityScoreThreshold=[0-1000]                                    // Optional. Your min priority score threshold 
```

Comment:
Why are we dropping this?

Reply:
It doesn't exist anymore, we only have score available now.